src/config.c

```diff
@@ -234,7 +234,7 @@ CONFIG_GETTER(getSearchThreads) {
 
 // WORKER_THREADS
 CONFIG_SETTER(setWorkThreads) {
-  int acrc = AC_GetSize(ac, &config->numWorkerThreads, AC_F_GE1);
+  int acrc = AC_GetSize(ac, &config->numWorkerThreads, AC_F_GE0);
```

Comment:
Still think this is meaningless and there is no use in giving the option to set it to 0. Can settle on a third opinion :)

Reply:
what if someone wants to load a server by taking the threads configuration parameters of another server?

Reply:
@meiravgri if WORKERS_THREAD is allowed to be 0, it actually means ENABLE_THREADS cannot be enabled, so it seems it should be greater or equal to 1 (unless we want this ability to disallow enabling threads in run-time)

Reply:
The code that decides if to go to the background or not checks that we are enabling it and that we have BG thread, so we won't crash if someone decides to enable the feature without any threads (but it will be meaningless)